includes/providers/gam/class-gam-scripts.php

```diff
@@ -242,7 +242,7 @@ function inOrPastViewport( element ) {
 						availableWidth = Math.max( boundsWidth, containerWidth ) + parseInt( ad_unit['bounds_bleed'] );
 						for ( viewportWidth in ad_unit['size_map'] ) {
 							var width = parseInt( viewportWidth );
-							if ( shouldUseBounds && width > availableWidth ) {
```

Comment:
Question: Is removing this `shouldUseBounds` intentional? Looks like this was checking the width of the container or something?

Reply:
It's intentional, notice that this variable is already checked on line 240/